terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/queryescape/QueryEscapeHelper.java

```diff
@@ -31,4 +46,29 @@ public void bindToModel(String searchPattern, List<Todo> todoList,
         model.addAttribute("todoList", todoList);
     }
 
+    public String getDatabaseId() {
+        return sqlSessionFactory.getConfiguration().getDatabaseId();

```

Comment:
This logic should be in `VersionInfoService` because `VersionInfoMyBatisServiceImpl` knows MyBatis3 is used.